### PR TITLE
[SENSOR][PYTHON][WEBGL] Equirectangular Bindings

### PR DESCRIPTION
**Pull request by Skylion007**
## Motivation and Context

* This PR adds python bindings for #1179 and improves existing bindings with the better hierarchy added in the PR.

## How Has This Been Tested
* Locally with PyTest

## Types of changes

- [X] New feature (non-breaking change which adds functionality)

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/esp/bindings/SensorBindings.cpp [hunk 1]
```diff
@@ -231,9 +240,25 @@ void initSensorBindings(py::module& m) {
           "far_plane_dist", &CameraSensor::getFar, &CameraSensor::setFar,
           R"(The distance to the far clipping plane for this CameraSensor uses.)");
 
+  // === CubeMapSensorBase ===
+  py::class_<CubeMapSensorBase,
+             Magnum::SceneGraph::PyFeature<CubeMapSensorBase>, VisualSensor,
+             Magnum::SceneGraph::PyFeatureHolder<CubeMapSensorBase>>(
+      m, "CubeMapSensorBase");
+
+  // === EquirectangularSensorSpec ===
```
Comment (bigbike) [inline src/esp/bindings/SensorBindings.cpp:249]: Typo in the comment: EquirectangularSensor

File: src/esp/bindings_js/bindings_js.cpp [hunk 2]
```diff
@@ -88,8 +91,8 @@ Magnum::Vector3 Vector3_sub(const Magnum::Vector3& v1,
 
 Observation Sensor_getObservation(Sensor& sensor, Simulator& sim) {
   Observation ret;
-  if (CameraSensor * camera{dynamic_cast<CameraSensor*>(&sensor)})
-    camera->getObservation(sim, ret);
+  if (VisualSensor * visSensor{dynamic_cast<VisualSensor*>(&sensor)})
```
Comment (bigbike) [inline src/esp/bindings_js/bindings_js.cpp:94]: Great.

**Review by bigbike — APPROVED** [review]
@erikwijmans : If possible, do you have time to take a pass? Thanks.

**Comment by bigbike** [comment]
This is great. Finally we can apply the fisheye sensor and equirectangular sensor in the training.

**Review by erikwijmans — APPROVED** [review]

**Pull request merged**